src/Discord.Net.Commands/Extensions/MessageExtensions.cs

```diff
@@ -45,7 +45,7 @@ public static bool HasStringPrefix(this IUserMessage msg, string str, ref int ar
         public static bool HasMentionPrefix(this IUserMessage msg, IUser user, ref int argPos)
         {
             var text = msg.Content;
-            if (text.Length <= 3 || text[0] != '<' || text[1] != '@') return false;
+            if (!string.IsNullOrEmpty(text) && (text.Length <= 3 || text[0] != '<' || text[1] != '@')) return false;
```

Comment:
My mistake, the logic for this is still incorrect. This returns `false` if any of these conditions are true, which is unlike the others, so `!string.IsNullOrEmpty(text)` isn't correct.
```suggestion
            if (string.IsNullOrEmpty(text) || text.Length <= 3 || text[0] != '<' || text[1] != '@') return false;
```

Reply:
Change Made.